lib/tarearbol/application.ex

```diff
@@ -23,6 +23,15 @@ defmodule Tarearbol.Application do
     Task.Supervisor.children(Tarearbol.Application)
   end
 
+  def jobs do
+    Tarearbol.Application.children
+    |> Enum.map(&Process.info/1)
```

Comment:
lol `&Process.info/2(:dictionary)` you’re overestimating the already existing syntax; with this PR please go to core team :)